src/Table.luna

```diff
@@ -863,13 +1356,180 @@ class Table:
         newTable
 
     def chart: ChartBuilder self
+
+    # Plots y versus x. Default style for plot is marker.
```

Comment:
what marker? It doesn't really tell me what the chart is going to look like.

Reply:
Also, if this is the default then there should be also a mention how to change it.